examples/tutorial/jupyter/execution/pandas_on_dask/test/test_notebooks.py

```diff
@@ -0,0 +1,117 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import os
+import sys
+
+import nbformat
+
+MODIN_DIR = os.path.abspath(
```

Comment:
What about just using this?
```python
import modin
modin.__file__
```

Reply:
I think usage of this approach can bring us additional complication - this approach strongly requires modin built from sources and only in development mode (pip command should be `pip install -e .[dask/ray]`).

Reply:
I see, makes sense.